src/core/bot/transit.py

```
@@ -0,0 +1,180 @@
+            Constants.log.info(
+                f'{message.message.group_openid} was banned to call {message_id.command}.')
+            raise UnauthorizedError("榜单功能被禁用")
+
+        try:
+            starts_with = message_id.command[-1] == '*' and func.startswith(message_id.command[:-1])
+            if starts_with:
+                name = message_id.command[:-1]
+                replaced = func.replace(name, '')
+                message.tokens = [name] + ([replaced] if replaced else []) + message.tokens[1:]
+            original_command(message)
+        except Exception as e:
```

Pylint (3.3.7)

[warning] Catching too general exception Exception

(W0718)